Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -mod=vendor -a -installsuffix
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -mod=vendor -a -installsuffix cgo -o dbinit cmd/dbinit/main.go
 
 FROM alpine:3.11
-RUN apk --no-cache add ca-certificates
+RUN apk --no-cache add ca-certificates tzdata
```

Comment:
We now import the `scrapedumper/pkg/postgres` package, which tries to load the timezone `America/New_York` by name. This'll fail unless the official IANA timezone databases are installed